components/components.css

```diff
@@ -186,3 +186,44 @@ iframe {
     border-radius: inherit;
     background: var(--sky);
 }
+
+input {
```

Comment:
Add parent class name here to avoid messing with other input tags
like this
.checkbox input {...}